apps/api-extractor/src/api/IConfigFile.ts

```diff
@@ -405,4 +406,15 @@ export interface IConfigFile {
    * to prevent spurious diffs in output files tracked for tests.
    */
   testMode?: boolean;
+
+  /**
+   * Specifies how API Extractor sorts the members of an enum when generating the .d.ts rollup and
+   * .api.json doc model.  The API report is always sorted to avoid spurious diffs.
```

Comment:
Now that we're sorting the API report, this comment needs to be updated.

Reply:
Added to mention report + markdown